ramls/circulation.raml

```diff
@@ -302,21 +302,21 @@ resourceTypes:
             description: "Shelving location id"
             pattern: "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$"
             required: true
-          loan_rules:
-            description: "Loan rules if provided, otherwise use stored loan rules"
+          circulation_rules:
```

Comment:
Maybe change this to rules as well?

Reply:
Changed in https://github.com/folio-org/mod-circulation/commit/4b0a55b49ecc627393e69877032f53a9554e7d7f